src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -135,6 +137,7 @@ private SamReaderFactoryImpl(final EnumSet<Option> enabledOptions, final Validat
             this.samRecordFactory = samRecordFactory;
             this.validationStringency = validationStringency;
             this.customReaderFactory = CustomReaderFactory.getInstance();
+            this.referenceSource = EMPTY_REFERENCE;
```

Comment:
@yfarjoun I think this will mask the fallback chain thats in place in the default reference source implementation (the code that uses environment variables to try the ENA download service or DEFAULT_FASTA settings - I don't think we have tests for those). Its currently established through the calls in SamReaderFactory to getDefaultCRAMReferenceSource. When we switched to using an interface based pattern for reference source for IGV, I intended to break those out into a separate implementations for each type (ENA ref service, DEFAULT_FASTA, user-supplied file) so we could establish which one to use once up front, but never did. For now they're still all mashed together in the one implementation.